packages/http-smoke-testing/.travis.yml

```diff
@@ -17,13 +17,14 @@ matrix:
             env: DEPENDENCIES="--prefer-lowest --prefer-stable"
         -   php: 7.1
             env: DEPENDENCIES="--prefer-lowest --prefer-stable"
+        -   php: 7.1
+            env: CHECK_STANDARDS="true"
 
 install:
     - travis_retry composer update $DEPENDENCIES
+    - if [[ "$CHECK_STANDARDS" = "true" ]]; then composer require --dev shopsys/coding-standards:dev-master; fi
```

Comment:
why dev-master ????
all branches and tags that are locaten in repo shopsys/http-smoke-testing are distributed into packagist
https://docs.travis-ci.com/user/environment-variables#default-environment-variables

Reply:
https://github.com/shopsys/http-smoke-testing/commit/ef64668575e51c350c9219cbfe0997ecf2adc207

Reply:
it seems also that some dependencies are missing.

Reply:
all packages are dependent on `shopsys/coding-standards:dev-master` (except in tagged commits, see `composer.json`)

Reply:
The problems with symfony/finder installation were fixed by 6224ef1f0cab934c9eee11f347cfcf7c35e3f1fb, see https://travis-ci.org/shopsys/http-smoke-testing/builds/531622657

For some reason, when the symfony/finder v4 was already installed, Composer didn't downgrade it to v3 so that shopsys/coding-standards can be installed - it just rejected the package requirement. I just switched the order of `composer` calls - I required the package before dependencies were installed which ended up just fine.